src/AWS.Deploy.Common/Recipes/RecipeDefinition.cs

```diff
@@ -28,6 +28,11 @@ public class RecipeDefinition
         /// </summary>
         public string Name { get; set; }
 
+        /// <summary>
+        /// Indicates if this recipe should be presented as an option during new deployments.
+        /// </summary>
+        public bool Hidden { get; set; }
```

Comment:
I know this is taken from my doc but I want if we should just be more explicit in the name instead of relying on comments to understand what it is for. Something like `DisableNewDeployments`.

Reply:
Fixed